demisto_sdk/common/constants.py

```diff
@@ -695,3 +695,13 @@ class PB_Status:
     'ip': {'IP.Address'},
     'url': {'URL.Data'}
 }
+
+KNOWN_WORDS = [
```

Comment:
we should have this as a file and read it, add words like Demisto, DBotScore please

Reply:
moved out of constants to a different file